README.md

```diff
@@ -17,6 +65,15 @@ cosmoz-moment is a Polymer component for centralized management of Moment.js wit
 <link rel="import" href="bower_components/cosmoz-moment/cosmoz-moment.html" />
 ```
 
+### Use it inside your element
+```html
+<cosmoz-moment locale="[[locale]]"></cosmoz-moment>
+```
+
 ## Docs
```

Comment:
l think we can leave it if we have plans to add more complex examples. This link to docs will show `gh-pages` with property list and some additional examples. `gh-pages` provide live docs and previews of our element via GitHub pages.